examples/client/client/tasks.nim

```diff
@@ -22,139 +22,101 @@ type
   StatusArg* = ref object of ContextArg
     clientConfig*: ClientConfig
 
-  StatusState* = enum loggedout, loggingin, loggedin, loggingout
-
-  WakuState* = enum stopped, starting, started, stopping
-
-const
-  DefaultTopic = waku_chat2.DefaultTopic
-  toyChat2App = "toy-chat/2"
-  waku1App = "waku/1"
+  UserMessageEventResult = Result[UserMessageEvent, string]
```

Comment:
I read this after my write-up in #287, where I proposed we go with `string` because the error is destined for display in the TUI.

But it does make a lot of sense to use enum. A step in the modified `dispatch` can then map from the enum values to an appropriate string, which is great because helps keep the client and TUI cleanly separated.